lms/envs/common.py

```diff
@@ -4701,6 +4701,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 # .. setting_description: Base URL of the micro-frontend-based dicussions page.
 # .. setting_warning: Also set site's courseware.discussions_mfe waffle flag.
 DISCUSSIONS_MICROFRONTEND_URL = None
+ENABLE_AUTHN_RESET_PASSWORD_HIBP_POLICY = False
```

Comment:
Please add the annotations for this flag as per https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html#boilerplate-template